### ISSUE TITLE
Pretrained cross-encoder/ms-marco* models produce values outside of [0,1]

### ISSUE THREAD

**Issue by ykhatami** [UKPLab/sentence-transformers#1546]
The pretrained cross-encoders on msmarco dataset produce output values which are outside of [0,1]. I can confirm it on "cross-encoder/ms-marco-MiniLM-L-12-v2", and others listed on https://www.sbert.net/docs/pretrained_cross-encoders.html.

For example, `model.predict([sentence1, sentence2])` returns -4.8. Models pretrained on STS and NLI seem fine.
Is this the expected behavior?

**Comment by nreimers** [comment]
It is expected. These model don't apply the sigmoid function and give you the raw logits.
